### DISCUSSION TITLE
More complete set of animations

### DISCUSSION THREAD

**Discussion by bijington** [xamarin/XamarinCommunityToolkit#1441] [Ideas]
I have recently implemented some animations based on this CSS site:
https://animate.style

Tada being a big favourite now (see attached)

https://user-images.githubusercontent.com/17139988/123698685-5896d880-d856-11eb-976c-b4f208e5b73b.mov

I wanted to investigate whether you would like a more complete set of animations?

My original approach has been to wrap these animations up inside extension methods so that they can easily be shared and triggered outside of user interaction. Would this approach suit XCT or would you prefer to keep them under the Behaviors approach and simply invoke them via a Command?

**Comment by matt-goldman** [comment]
These look great and I would definitely find this useful. For my 2c worth, having the option of triggering them through user interaction for from a code event would be awesome. For example, animating a little bell icon to indicate a notification.

**Reply by bijington** [reply]
Thanks yes this makes perfect sense. That part I quite like from my own internal implementation at the moment is my extension method can be applied to multiple controls in one go, I don't think there is a way to do this with the existing AnimationBehavior parts in XCT. I suspect I would offer up the extension methods and then have the AnimationBehavior call the extension methods so we get more support throughout.

**Comment by AndreiMisiukevich** [comment]
From my perspective it looks great) nice job btw

**Reply by bijington** [reply]
Thank you 😄 . I will go ahead and raised a feature then and try to propose the full API changes

**Comment by ghost** [comment]
Nice job 👌

**Comment by jbachelor** [comment]
This would be an absolutely wonderful thing to add... I love the Tada sample! Thank you so much for being willing to share your hard work!
